doc/sphinxext/related_software.py

```diff
@@ -97,11 +99,6 @@
         "Home-page": "https://emd.readthedocs.io/en/stable",
         "Summary": "Empirical Mode Decomposition in Python.",
     },
-    # Needs https://github.com/vferat/pycrostates/pull/188 and a release
```

Comment:
... but either way we should keep this PR open so we can benefit from this simplification